solr/app.py

```diff
@@ -17,10 +19,37 @@ def create_app():
     app.url_map.strict_slashes = False
     Consul(app)  # load_config expects consul to be registered
     load_config(app)
+    if config:
+        app.config.update(config)
+        
+    db.init_app(app)
     logging.config.dictConfig(
         app.config['SOLR_SERVICE_LOGGING']
     )
     
+    ## pysqlite driver breaks transactions, we have to apply some hacks as per
```

Comment:
Indeed, sqlite *can* be used in production. And during development it should work the same. You are making assumption that we always use postgres, but that is not correct. Actually, solr-service is used by the guys in Chicago and the fact they don't have to depend on postgres is exactly needed. 

But in general, from the start on, we have decided to not depend on one database and that should stay - unless of course there are strong reasons to abandon that strategy.